src/chrome/content/rules/DuckDuckGo.xml

```diff
@@ -41,24 +34,35 @@
 	<target host="ac.duckduckgo.com" />
 	<target host="api.duckduckgo.com" />
 	<target host="beta.duckduckgo.com" />
+	<target host="blog.duckduckgo.com" />
 	<target host="bttf.duckduckgo.com" />
 	<target host="chrome.duckduckgo.com" />
 	<target host="dub.duckduckgo.com" />
+	<target host="favicons.duckduckgo.com" />
+		<test url="http://favicons.duckduckgo.com/ip2/twitter.com.ico" />
+		<exclusion pattern="^http://favicons\.duckduckgo\.com/$" />
 	<target host="ff.duckduckgo.com" />
+	<target host="help.duckduckgo.com" />
 	<target host="i.duckduckgo.com" />
 	<target host="images.duckduckgo.com" />
+	<target host="icons.duckduckgo.com" />
+		<test url="http://icons.duckduckgo.com/ip2/twitter.com.ico" />
+		<exclusion pattern="^http://icons\.duckduckgo\.com/$" />
```

Comment:
Why is the exclusion here required?

Reply:
Strange. When I tried the two domains yesterday it resulted in a 500 error. Know it shows an search interface which results in a 502 when used to search. :/